x-pack/plugins/cloud_security_posture/public/components/csp_page_template.tsx

```diff
@@ -91,6 +91,7 @@ const getPackageNotInstalledNoDataConfig = (
   actions: {
     elasticAgent: {
       href: cisIntegrationLink,
+      isDisabled: !cisIntegrationLink,
```

Comment:
we need the integration name and version to generate the link
we get that from an API 

so until we have the link, the buttons that show it are disabled

Reply:
a thought - while showing it as disable, showing a spinner inside of it.
So a user would have an idea that that is being calculated

Reply:
The rendered component is not totally in our control so not sure how simple this will be, but a good idea any way... not sure regarding scope right now

Reply:
I think in reality this will never happen because this UI is only presented after we check if the integration is installed or not, so the query will be cached in all cases